src/main/java/kr/syeyoung/dungeonsguide/dungeon/actions/ActionBreakWithSuperBoom.java

```diff
@@ -39,26 +40,17 @@
 import java.util.Set;
 
 @Data
+@EqualsAndHashCode(callSuper=false)
 public class ActionBreakWithSuperBoom extends AbstractAction {
-    private Set<Action> preRequisite = new HashSet<Action>();
+    private Set<AbstractAction> preRequisite = new HashSet<AbstractAction>();
     private OffsetPoint target;
 
     public ActionBreakWithSuperBoom(OffsetPoint target) {
         this.target = target;
     }
 
```

Comment:
Prerequistes and isComplete?

Reply:
idk what you mean, i mean it works. i just flattened a interface that had only one implementation

Reply:
You didn't implement prerequisites and isComplete method